src/viam/app/data_client.py

```diff
@@ -1474,7 +1490,7 @@ async def streaming_data_capture_upload(
         data_request_times: Optional[Tuple[datetime, datetime]] = None,
         tags: Optional[List[str]] = None,
     ) -> str:
-        """Uploads the metadata and contents of streaming binary data.
+        """Uploads the metadata and contents of streaming binary data to app.viam.com.
```

Comment:
I think technically a user could pass a separate app url and so use something other than `app.viam.com` (say, e.g., someone wants to have a client to `app.viam.dev`). So I don't think we should be too specific here.

edit: looks like `app.viam.com` is hardcoded elsewhere here. Unfortunate! Given that, my slight preference is to still not update here (and possibly remove in other locations?). But I suppose it's not a huge deal, given that this has existed for a while already.

Reply:
I actually wanted to remove this elsewhere, but decided to leave the mentions in place because I didn't want to change too much at once... glad we're on the same page! I'll remove all of these mentions, which are 99% redundant.